.eslintrc

```diff
@@ -54,8 +54,10 @@
     "no-multi-assign": "off",
     "react/jsx-closing-tag-location": "error",
     "react/no-unused-state": "error",
-    "react/prefer-stateless-function": "error"
+    "react/prefer-stateless-function": "error",
 
+    // Errors Expected linebreaks to be 'LF' but found 'CRLF'
+    "linebreak-style": ["error", "windows"]
```

Comment:
@alreadyExisted This'll will cause a lot of errors in continuous-integration test.